common/messages/messages_en.go

```diff
@@ -182,7 +168,7 @@ var messagesEn = map[string]util.Message{
 		Description: "A subnet with the specified zone '%s' and available cluster subnet list '%s' could not be found.",
 		Type:        util.RetrivalFailed,
 		RC:          404,
-		Action:      "Verify that the subnet from the cluster subnet list exists. Target to appropriate region 'ibmcloud target -r <region>' and verify if 'ibmcloud is subnets' is returning the subnets. If it is not returning the matching subnet from the cluster subnet list then raise ticket for VPC team else raise ticket for IKS team.",
+		Action:      "Please verify if VPC and Cluster are part of the same ResourceGroup. Please refer https://cloud.ibm.com/docs/containers?topic=containers-storage-file-vpc-apps#storage-file-vpc-custom-sc for more details and if required raise ticket for IKS team.",
```

Comment:
same here

Reply:
If VPC and Cluster are part of the different ResourceGroup. Please refer https://cloud.ibm.com/docs/containers?topic=containers-storage-file-vpc-apps#storage-file-vpc-custom-sc for more details and if required raise ticket for IKS team. If it is not then raise ticket for IKS team.

Reply:
Done @derekpoindexter suggestions incorporated